deployment/ansible/development/roles/pycharm/templates/workspace-configuration.xml.j2

```diff
@@ -387,6 +385,27 @@
       <option name="SHOW_COMMAND_LINE" value="false" />
       <method />
     </configuration>
+    <configuration name="Run Realtime Tests" type="PythonConfigurationType" factoryName="Python">
```

Comment:
I added another Run Configuration specifically to run Realtime tests.
This is because I have issues with PyCharm's Django tests didn't start liveserver when I'm using LiveServerTestCase.